org.mwc.debrief.legacy/src/Debrief/Wrappers/TrackWrapper.java

```diff
@@ -912,13 +913,13 @@ public static int mergeTracksInPlace(final Editable target, final Layers theLaye
 	 * the formatting details
 	 */
 	public TrackWrapper() {
-		_mySensors = new SplittableLayer(true);
+		_mySensors = new RestrictedBaseLayer<SensorWrapper>(true);
 		_mySensors.setName(SENSORS_LAYER_NAME);
 
 		_myDynamicShapes = new SplittableLayer(true);
 		_myDynamicShapes.setName(DYNAMIC_SHAPES_LAYER_NAME);
 
-		_mySolutions = new BaseLayer(true);
+		_mySolutions = new RestrictedBaseLayer<>(true);

```

Comment:
`_mySolutions` can be restricted to instances of TMAWrapper

Reply:
done